maven-repository-provisioner/src/main/java/com/simpligility/maven/provisioner/PackagingChecker.java

```diff
@@ -0,0 +1,37 @@
+package com.simpligility.maven.provisioner;
+
+import java.util.List;
+
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.artifact.DefaultArtifact;
+
+public class PackagingChecker 
+     {
+
+    private static final String POM = "pom";
+
+    private static final String JAR = "jar";
+
+
+    public static void filter( 
```

Comment:
maybe postProcess(...) rather than filter... since you may add or remove items to be downloaded